packages/mgt-spfx-utils/README.md

```diff
@@ -0,0 +1,155 @@
+# Utilities for SharePoint Framework Web Parts using Microsoft Graph Toolkit
+
+[![npm](https://img.shields.io/npm/v/@microsoft/mgt-spfx-utils?style=for-the-badge)](https://www.npmjs.com/package/@microsoft/mgt-spfx-utils)
+
+![SPFx 1.15.2](https://img.shields.io/badge/SPFx-1.15.2-green.svg?style=for-the-badge)
+
+Helper functions to simplify lazy loading of Microsoft Graph Toolkit components when using disambiguated web components in SharePoint Framework web parts. For more information on disambiguation in MGT see https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/main/packages/mgt-components#disambiguation
+
+## Installation
+
+To lazy load Microsoft Graph Toolkit components from the library, add the `@microsoft/mgt-spfx-utils` package and if using React, the `@microsoft/mgt-react` package as dependencies to your SharePoint Framework project:
+
+```bash
+npm install @microsoft/mgt-spfx-utils
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils
+```
+
+or when using React:
+
+```bash
+npm install @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+> **Important:** Since a given web component tag can only be registered once these approaches **must** be used along with the `customElementHelper.withDisambiguation('foo')` approach as this allows developers to create disambiguated tag names.
+
+By disambiguating tag names of Microsoft Graph Toolkit components, developers can use their own version of MGT rather than using the centrally deployed `@microsoft/mgt-spfx` package. This allows them to avoid colliding with other SharePoint Framework components, built by other developers. When disambiguating tag names, MGT is included in the SPFx bundle, increasing its size.
+
+## Usage
+
+### When using no framework web parts
+
+When building SharePoint Framework web parts without a JavaScript framework the `@microsoft/mgt-components` library must be asynchronously loaded after configuring the disambiguation setting. The `importMgtComponentsLibrary` helper function wraps this functionality. Once the `@microsoft/mgt-components` library is loaded you can load components directly in your web part.
+Below is a minimal example web part that demonstrates how to use MGT with disambiguation in SharePoint Framework Web parts. A more complete example is available in the [No Framework Web Part Sample](https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/main/samples/sp-mgt/src/webparts/helloWorld/HelloWorldWebPart.ts).
+
+```ts
```

Comment:
This is a great complete sample that shows how to use disambiguation in MGT with SPFx 👏